validator/client/src/main/java/tech/pegasys/teku/validator/client/signer/BlockContainerSignerDeneb.java

```diff
@@ -44,15 +49,21 @@ public SafeFuture<SignedBlockContainer> sign(
     return signBlock(unsignedBlock, validator, forkInfo)
         .thenApply(
             signedBlock -> {
-              // Blinded flow
```

Comment:
The flow for creating a block for Deneb is a bit weird, so in case for blinded, we just return the `SignedBeaconBlock` as it is after signing it. For non blinded flow, we create the block contents and they should always have proofs and blobs, so I think it's better to be more harsh and use `orElseThrow()` and throw an exception.

Reply:
Alright, I pushed some changes, is that what you meant?

Reply:
Yup. It looks ok.

Reply:
Cool, I'll add the missing tests then